ROADMAP.md

```diff
@@ -0,0 +1,28 @@
+# Material Web Components Roadmap
```

Comment:
I kinda have a feeling this file is going to get out of date. What's to keep that from happening? Why don't we just make issues for anything that's missing?

I could imagine an internal google doc, with more long term planning, and where people can easily comment etc., could be useful, but not sure about this checked in here.

Reply:
Hmm, maybe the README can already count as enough of a Roadmap, now that we have the status of all the components in there.

I can make issues for the other topics.

Reply:
Created #523 for updating slider design, #524 for updating linear-progress design, #525 for reviewing material studies, and #526 for exploring theme systems.